grammar.js

```diff
@@ -1002,6 +1004,20 @@ module.exports = grammar({
       )
     },
 
+    include: $ => {
+      return seq(
+        'include',
+        $._type
```

Comment:
Part of me is worried this is too generic / captures too much

Reply:
This should only allow `constant`, `generic_instance_type` and `self`, same with `extend`.